exchange/exchange_test.go

```diff
@@ -2297,6 +2297,8 @@ func runSpec(t *testing.T, filename string, spec *exchangeSpec) {
 			assert.JSONEq(t, expectedPassthough, actualPassthrough, "Expected bid response extension is incorrect")
 		}
 
+	} else if spec.Response.Ext != nil {
+		assert.JSONEq(t, string(spec.Response.Ext), string(bid.Ext), "ext mismatch")
```

Comment:
This unconditional `Response.Ext` check uncovered a "false pass" in the json tests added with #2450 
`spec.BidValidationEnforcement` is never set so the section below does not run, and now the `ext` check I added picked up on the difference. I'll add a comment on that PR.

After the fixes we will still have three separate places that check `ext` - bid validation, passthrough and fledge. Would be nice to consolidate - the issue for me was the ever-present "debugging disabled for account" message would require touching too many unrelated (for me) files to fix.

Reply:
Hey @laurb9 I just opened a PR https://github.com/prebid/prebid-server/pull/2529, to address the false pass in relation to `spec.BidValidationEnforcement`